conseil-api/src/main/scala/tech/cryptonomic/conseil/api/config/ConseilAppConfig.scala

```diff
@@ -49,8 +49,6 @@ trait ConseilAppConfig {
   protected def loadApplicationConfiguration(commandLineArgs: Array[String]): ConfigReader.Result[Configurations] = {
     import ConseilAppConfig.Implicits._
 
-    implicit def hint[T]: ProductHint[T] = ProductHint[T](ConfigFieldMapping(CamelCase, CamelCase))
```

Comment:
Are you sure we can remove it?

The compiler flagged this as unused in the past, but in fact it might break the actual runtime behaviour.
Do the tests run correctly even without this hint?

Reply:
Yes, tests are passing. I'm quite positive about that, because only `tech.cryptonomic.conseil.api.config.ConseilAppConfig.Implicits#tezosConfigurationReader`requires that, according to tests and what I've noticed in configuration itself.

Yes, compiler flags that, but when I removed that from class mentioned above, tests are failing, here, nothing happens.

Reply:
I just hope we're not breaking things at runtime, that we didn't manage to unit-test 😅